code/modules/mod/mod_construction.dm

```diff
@@ -0,0 +1,187 @@
+/obj/item/mod/construction
+	desc = "A part used in MOD construction."
+
+/obj/item/mod/construction/helmet
+	name = "MOD helmet"
+	icon_state = "helmet"
+
+/obj/item/mod/construction/chestplate
+	name = "MOD chestplate"
+	icon_state = "chestplate"
+
+/obj/item/mod/construction/gauntlets
+	name = "MOD gauntlets"
+	icon_state = "gauntlets"
+
+/obj/item/mod/construction/boots
+	name = "MOD boots"
+	icon_state = "boots"
+
+/obj/item/mod/construction/core
+	name = "MOD core"
+	icon_state = "mod-core"
+	desc = "A mystical crystal able to convert cell power into energy usable by MODsuits."
+
+/obj/item/mod/construction/armor
+	name = "MOD standard armor plates"
+	desc = "Armor plates used to finish a MOD"
+	icon_state = "armor"
+	var/theme = /datum/mod_theme
+
+/obj/item/mod/construction/armor/engineering
+	name = "MOD engineering armor plates"
+	icon_state = "engineering-armor"
+	theme = /datum/mod_theme/engineering
+
+/obj/item/mod/paint
+	name = "MOD paint kit"
+	desc = "This kit will repaint your MODsuit to something unique."
+	icon = 'icons/obj/mod.dmi'
+	icon_state = "paintkit"
+
+/obj/item/mod/construction/shell
+	name = "MOD shell"
+	icon_state = "mod-construction"
+	desc = "An empty MOD shell."
+	var/obj/item/core
+	var/screwed_core = FALSE
+	var/obj/item/helmet
+	var/obj/item/chestplate
+	var/obj/item/gauntlets
+	var/obj/item/boots
```

Comment:
Make sure you properly qdel all these.

Reply:
Also, add a `/handle_atom_del` implementation so that if they delete inside the shell for whatever reason, they don't hard del.